docs/best_practices/tool_xml.rst

```diff
@@ -21,6 +21,24 @@ attribute of the XML tool element, e.g.
 and should be incremented with each change of the wrapper that is
 released to the Tool Shed (except for cosmetic modifications).
 
+The value should follow the
+`PEP 440 <https://www.python.org/dev/peps/pep-0440/>`__ specification.
+
+If the Galaxy tool is a wrapper for an underlying tool, we recommend to set the
+``version`` attribute to something like ``@TOOL_VERSION@+galaxyN``, where:
```

Comment:
Should we start adding the `+galaxy0` local version identifier only after an initial `@TOOL_VERSION@` release?
It seems @bgruening was suggesting this in https://github.com/galaxyproject/tools-iuc/pull/1800#discussion_r177648287

Reply:
I thought that was only in the context of a pre-existing wrapper already using ``@TOOL_VERSION`` which would start to use ``@TOOL_VERSION+galaxyN`` style without messing up version sorting.

Generally a new tool would start at ``@TOOL_VERSION+galaxy0`` (wouldn't it?)

Reply:
No hard feelings here. I would like to see something like `@TOOL_VERSION@@WRAPPER_VERSION@` if WRAPPER_VERSION default to `   ` or to `+galaxy0` for the first version I do not know. I would slightly prefer the `    ` version, but only for esthetically reasons in the Galaxy tool user interface.

Reply:
I've reworded the text to also allow just `@TOOL_VERSION@` for the first wrapper release of each version of the underlying tool.

Reply:
OK. I prefer the explicit ``+galaxy0`` but the wording proposed by Nicola is fine.